### DISCUSSION TITLE
Launch control screen doesn't show anything

### DISCUSSION THREAD

**Discussion by banklo** [cedricp/ddt4all#1178] [General]
Hey guys, a was relearning my clutches on clio iv rs and wandered into Launch Control screen and it's blank. Any idea what may be wrong? Launch control works normally...

`Traceback (most recent call last):
  File "C:\Python27\lib\lib-tk\Tkinter.py", line 1547, in __call__
    return self.func(*args)
  File "C:\Users\thinkpad\Documents\pyren-master/pyren\mod_ddt_screen.py", line 889, in OnTreeClick
    self.loadScreen (self.Screens[iid])
  File "C:\Users\thinkpad\Documents\pyren-master/pyren\mod_ddt_screen.py", line 1441, in loadScreen
    self.images.append (tk.PhotoImage (file=gifname))
  File "C:\Python27\lib\lib-tk\Tkinter.py", line 3377, in __init__
    Image.__init__(self, 'photo', name, cnf, master, **kw)
  File "C:\Python27\lib\lib-tk\Tkinter.py", line 3331, in __init__
    self.tk.call(('image', 'create', imgtype, name,) + options)
TclError: error reading color map
`

![Capture](https://github.com/user-attachments/assets/b71952eb-8635-4bd8-8e2f-6b70cccc1a2c)

**Comment by AndyIreland** [comment]
your screenshot is of PyREN thats on 
https://gitlab.com/py_ren/pyren/-/issues

but if you do go to DDT4ALL program it is possible that you could manually change settings with this button 
![image](https://github.com/user-attachments/assets/692d7288-88d9-4f71-bf1f-4b2e7b355eb5)
  - however could be really dangerous I would prefer to not make adjustments that way personally.
  it just dont look in the database (DDT2000) the graphics have been filled out properly in the GUI

**Reply by banklo** [reply]
Oh sorry and thank you!
